chapter2/tests/t16/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Task 11 - slow php reporting page
+set -eo pipefail
+
+# Start php-fpm in foreground mode
+php-fpm -F --fpm-config $(pwd)/.php.fpm.d/php-fpm.conf &
```

Comment:
I don't see .php.fpm.d folder. 
your php-fpm.conf should be placed in a proper place so the php-fpm buildpack kicks in.

Reply:
See more:
https://github.com/paketo-buildpacks/samples/tree/main/php

Reply:
your root level php-fpm.conf should include *.conf files in the .php.fpm.d folder that define slow.conf and fast.conf pools

Reply:
> I don't see .php.fpm.d folder. your php-fpm.conf should be placed in a proper place so the php-fpm buildpack kicks in.

@dmitry-mightydevops 
Done